index.html

```diff
@@ -1557,6 +1557,27 @@ <h2 id="sgHeader" class="accordion-header">
                     </div>
                 </form>
             </div>
+            <div class="modal" tabindex="-1" id="confirmDeleteRoundModal">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title">Delete Round?</h5>
+                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```

Comment:
Can you add the "aria-label" to the other buttons as well to add accessibility to the app.